src/onboarding2/Deployment/Deployment.js

```diff
@@ -20,6 +21,14 @@ import allDoneImg from './assets/illustration-all-done.png'
 const AnimDiv = animated.div
 const AnimSection = animated.section
 
+const TransactionsStatusType = PropTypes.arrayOf(
+  PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    status: PropTypes.oneOf(['error', 'pending', 'success', 'upcoming'])
```

Comment:
Same question about `Symbol`s, if the actual hardcoded value is not being rendered we should `Symbol`ize these no?

Reply:
Yes we could do use symbols here as well :+1: